sub/source/docs/internal-leak-failsafe.rst

```diff
@@ -16,6 +16,8 @@ Whether the active level of an analog sensor or the logic level from a digital s
 
 .. note:: In ArduSub firmware prior or equal to 4.5, the input pin must be declared to be analog or digital with the ``LEAKx_TYPE`` parameter.
 
+.. note:: Digital leak detectors connected to an :ref:`AUX port <main-aux-out>` must be configured as :ref:`common-gpios`.
```

Comment:
doesnt this also apply to main outputs as well?

Reply:
I think I've come up with a reasonable compromise. Let me know if I've misused any of the terminology :-)